homeassistant/components/hausbus/config_flow.py

```diff
@@ -0,0 +1,43 @@
+"""Config flow for Haus-Bus integration."""
+from __future__ import annotations
+
+import logging
+from typing import Any
+
+import voluptuous as vol  # type: ignore[import]
+
+from homeassistant import config_entries
+from homeassistant.data_entry_flow import FlowResult
+from homeassistant.exceptions import HomeAssistantError
+
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+STEP_USER_SCHEMA = vol.Schema({})
+
+
+class ConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
+    """Handle a config flow for hausbus."""
+
+    VERSION = 1
+
+    async def async_step_user(
+        self, user_input: dict[str, Any] | None = None
+    ) -> FlowResult:
+        """Handle the initial step."""
+        errors: dict[str, str] = {}
```

Comment:
```suggestion
```
it seems like you're not doing anything here, so no errors possible. But I would argue that I'd make sense to check if we even have devices here first.

Reply:
Devices are added asynchronously. We are just setting things up and sending a UDP broadcast to the network for discovery afterwards. So at this step we do not have any devices.